### DISCUSSION TITLE
Lista clienti con Python SDK: address_street sempre vuoto e ottenere solo fields desiderati non funziona

### DISCUSSION THREAD

**Discussion by studio121-develop** [fattureincloud/api#415] [Q&A] [answered]
Buongiorno,

ho creato uno script python utilizzando l'Python SDK che ottiene la lista dei clienti e al momento la salva su un file txt.

Andando a visionare il file mi sono reso conto di almeno due cose:

1. tutti i field "address_street" sono None, se invece simulo da https://developers.fattureincloud.it/api-reference trovo gli effettivi indirizzi per tutti i clienti per i quali questo è indicato
2. da script se provo a passare api_response = api_instance.list_clients(company_id, fields=fields) mi riporta tutti i fields per ogni cliente

`# Enter a context with an instance of the API client
with fattureincloud_python_sdk.ApiClient(configuration) as api_client:
	# Create an instance of the API class
	api_instance = fattureincloud_python_sdk.ClientsApi(api_client)
	company_id = 1212121 # int | The ID of the company.
	fields = 'name' # str | List of comma-separated fields. (optional)

	try:
		# List Clients
		api_response = api_instance.list_clients(company_id, fields=fields)
		# Apertura (o creazione se non esiste) di un file in modalità scrittura
		with open("api_response.txt", "w") as file:
			print("Salvataggio della risposta di ClientsApi->list_clients nel file 'api_response.txt'")
			# Formattazione della risposta per una migliore leggibilità e scrittura nel file
			file.write(pformat(api_response))
	except ApiException as e:
		print(f"Exception when calling ClientsApi->list_clients: {e}\n")
		# In caso di eccezione, scrive l'errore nel file
		with open("api_response.txt", "w") as file:
			file.write(f"Exception when calling ClientsApi->list_clients: {e}\n")`

**Comment by emajo** [comment, marked as answer]
Ciao @studio121-develop, per recuperare il campo `address_street` è necessario specificare il `fieldset=detailed`, per quanto riguarda il comportamento del campo fields è molto strano, faremo dei controlli.

Grazie,
Emanuele.

**Reply by valmoz** [reply]
Buongiorno @studio121-develop 
è riuscito a risolvere il problema?

Grazie
Mauro
